src/renderers/testing/ReactTestRendererFiber.js

```diff
@@ -246,8 +193,18 @@ var TestRenderer = ReactFiberReconciler({
   useSyncScheduling: true,
 
   getPublicInstance(ref) {
-    const createNodeMock = ref.rootContainerInstance.createNodeMock;
-    return createNodeMock(ref);
+    if (typeof ref.text === 'string') {
```

Comment:
how would that look? I could add `$$typeof` symbols to the container and text instances..